codechat_config.yaml

```diff
@@ -34,7 +34,7 @@ output_path: output/web
 # - ``{output_path}``: the ``output_path`` above, but as an absolute path.
 # - ``{sys_executable}``: the value of the running Python's `sys.executable <https://docs.python.org/3/library/sys.html#sys.executable>`_.
 # - ``{xml_id}``: The value of the the first ``xml:id`` attribute in the currently open file.
-args: "{sys_executable} -m pretext build web --xmlid {xml_id}"
+args: pretext build web --xmlid {xml_id}
```

Comment:
:eyes:

I assume the old line was generated by PreTeXt. But it doesn't work?

Reply:
I believe this change was unnecessary (but probably harmless) - Codechat works fine for me on commit 47cb25e6ae9740e9f1b36dce2d938524d5fca228

Reply:
I think I may have mixed up some branches that was causing the issue, and resolving that branch confusion fixed it.

So this puts it back, and it seems to still be working for me 🤷🏼‍♂️ https://github.com/TeamBasedInquiryLearning/precalculus/pull/5